bin/desimodel_sync_kpno.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# This script is intended to be run from a cronjob (see 
+# desimodel/etc/desimodel_sync_kpno_cron.sh).  You should not run
+# this script interactively.  It expects to be run inside an
+# already-configured software environment.
+
+fpsync=$(which desi_sync_focalplane)
+echo "Using focalplane sync script:  ${fpsync}"
+
+# Find the newest calibration file
+caldir="/data/focalplane/calibration/"
+calfile=$(ls ${caldir} | egrep '[0-9]{8}T[0-9]{6}.*' | sort | tail -n 1)
+calpath="${caldir}/${calfile}"
+
+# Run it.
+eval ${fpsync} --calib_file ${calpath} --commit
```

Comment:
I'm not sure how easy this is from KPNO, but let's catch whether this encounters an error and proactively email people so that update failures don't silently disappear into a log we aren't actively monitoring.

Feature creep: in the short term, it would be reassuring to get an email for every update confirming
1. script ran but no updates were needed
2. script ran and updates were made (maybe with basic stats of N fibers changed state, e.g. for humans to catch if 300 fibers were unexpectedly disabled in the latest dump)
3. script ran and hit and error and needs attention

Reply:
Sounds good- is there an established protocol for sending email from KPNO systems as the datasystems user?  (i.e. SMTP smarthost that will relay mail).  Another option I can do without dealing with email permissions is to post (with curl) a message to a dedicated desi slack channel.

Reply:
@weaverba137 @elliotta do we have hooks setup for scripts sending emails from KPNO machines?  @tskisner posting to #survey-ops Slack wouldn't be bad either if you already know how to sort out the authentication stuff without bringing in additional dependencies.

FTR: I post-facto saw your second comment to the PR making basically the same suggestion to email /alert about failures, so apologies for typing the review as if the idea was new.

Reply:
For slack notifications, this is just outbound https post commands with curl, so no KPNO permissions needed.

Reply:
I assume it is _possible_ to send email from KPNO machines, because the night log has to get out somehow, but I don't know if there are any specific hooks or procedures that need to be followed.

Reply:
ok, let's be all modern and try the posting-to-slack route.